### PR TITLE
Change when checking for updates

### PR DESCRIPTION
**Pull request by erikjv**
We now handle user-requested actions from the command-line, and not check for updates, because a user would not appreciate it if the application quits because an update is found.

Also move opening the new-account wizard to after the update check, so that when no account is configured, and an update is available, only one dialog will be shown.

Fixes: #11141

### CODE REVIEW

**Review by TheOneRing — APPROVED** [review]

**Comment by TheOneRing** [comment]
Hmm it appears  to break some ui tests, the wizard does not show up.

File: src/gui/main.cpp [hunk 1]
```diff
@@ -417,24 +412,34 @@ int main(int argc, char **argv)
...
-
-#ifdef WITH_AUTO_UPDATER
-    // if handleStartup returns true, main()
-    // needs to terminate here, e.g. because
-    // the updater is triggered
-    Updater *updater = Updater::instance();
-    if (updater && updater->handleStartup()) {
-        return 1;
+    // Display the wizard if we don't have an account yet, and no other UI is showing.
+    if (!options.show && AccountManager::instance()->accounts().isEmpty()) {
```
Comment (TheOneRing) [inline src/gui/main.cpp:440]: why: `!options.show` ?
This appears to break the gui tests.
options.show will only bring the application to the front, which will be done by the wizard anyhow.

**Review by TheOneRing — CHANGES_REQUESTED** [review]

File: src/gui/main.cpp (hunk 1 above)
Comment (erikjv) [reply src/gui/main.cpp]: Ok, removed that part. Let's see how the tests do now.

**Comment by TheOneRing** [comment]
Hmm all tests failed

**Comment by TheOneRing** [comment]
I updated the branch lets see

**Pull request merged**